app/controllers/submissions_controller.rb

```diff
@@ -9,7 +9,11 @@ def new
   def create
     @submission.submitter = current_user
     if @submission.valid?
-      if @submission.passed_screening?
+      if @submission.followup? # this isn't working :(
```

Comment:
@mikedaul Is it now working and that comment was meant to be removed?

Reply:
Doh - thank you Sean! I'll clean that up (it should indeed be working)